Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -102,6 +102,10 @@ static void DSPCallback(u64 userdata, s64 cyclesLate)
 static void AudioDMACallback(u64 userdata, s64 cyclesLate)
 {
 	int period = s_cpu_core_clock / (AudioInterface::GetAIDSampleRate() * 4 / 32);
+	if (SConfig::GetInstance().bEmulationSpeedFixAudio &&
```

Comment:
Actually, with that commented out, and mine disabled, it sounds 100% correct when reaching full 200% FPS, but stutters before then.
With that commented out, and mine enabled, audio plays faster than normal speed, but never stutters.

Any way to edit a pull request?  And sorry, I don't know much about dolphin's internals, and where I edited is what I managed to find with Ctrl-F and Ctrl-Click